src/AnalysisPrograms/Recognizers/Birds/BotaurusPoiciloptilus.cs

```diff
@@ -0,0 +1,168 @@
+// <copyright file="BotaurusPoiciloptilus.cs" company="QutEcoacoustics">
+// All code in this file and all associated files are the copyright and property of the QUT Ecoacoustics Research Group (formerly MQUTeR, and formerly QUT Bioacoustics Research Group).
+// </copyright>
+
+namespace AnalysisPrograms.Recognizers
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Reflection;
+    using System.Runtime.CompilerServices;
+    using Acoustics.Shared.ConfigFile;
+    using AnalysisBase;
+    using AnalysisPrograms.Recognizers.Base;
+    using AudioAnalysisTools;
+    using AudioAnalysisTools.DSP;
+    using AudioAnalysisTools.Events;
+    using AudioAnalysisTools.Events.Types;
+    using AudioAnalysisTools.Indices;
+    using AudioAnalysisTools.StandardSpectrograms;
+    using AudioAnalysisTools.WavTools;
+    using log4net;
+    using SixLabors.ImageSharp;
+    using TowseyLibrary;
+    using static AnalysisPrograms.Recognizers.GenericRecognizer;
+    using Path = System.IO.Path;
+
+    /// <summary>
+    /// A recognizer for the Australasian Bittern, Botaurus poiciloptilus, https://en.wikipedia.org/wiki/Australasian_bittern.
+    /// The Australasian bittern, also known as the brown bittern or matuku hūrepo, is a large bird in the heron family Ardeidae.
+    /// A secretive bird with a distinctive booming call, it is more often heard than seen.
+    /// Australasian bitterns are endangered in both Australia and New Zealand.
+    /// </summary>
+    internal class BotaurusPoiciloptilus : RecognizerBase
+    {
+        private static readonly ILog BitternLog = LogManager.GetLogger(MethodBase.GetCurrentMethod().DeclaringType);
+
+        public override string Author => "Towsey";
+
+        public override string SpeciesName => "BotaurusPoiciloptilus";
+
+        public override string Description => "[ALPHA] Detects acoustic events for the Australasian Bittern.";
+
+        public override AnalyzerConfig ParseConfig(FileInfo file)
+        {
+            RuntimeHelpers.RunClassConstructor(typeof(BotaurusPoiciloptilusConfig).TypeHandle);
+            var config = ConfigFile.Deserialize<BotaurusPoiciloptilusConfig>(file);
+
+            // validation of configs can be done here
+            GenericRecognizer.ValidateProfileTagsMatchAlgorithms(config.Profiles, file);
+
+            // This call sets a restriction so that only one generic algorithm is used.
+            // CHANGE this to accept multiple generic algorithms as required.
+            //if (result.Profiles.SingleOrDefault() is OnebinTrackParameters)
+            if (config.Profiles?.Count == 1 && config.Profiles.First().Value is OnebinTrackParameters)
+            {
+                return config;
+            }
+
+            throw new ConfigFileException("Autralasian Bittern expects one and only one ForwardTrack algorithm.", file);
```

Comment:
```suggestion
            throw new ConfigFileException($"Autralasian Bittern expects one and only one {nameof(OneBinTrackParameters)}} algorithm.", file);
```

Reply:
I have fixed the class name. Suggestion done.